app/carbonapi/http_handlers.go

```diff
@@ -658,7 +661,7 @@ func (app *App) findHandler(w http.ResponseWriter, r *http.Request) {
 
 			http.Error(w, msg, code)
 			apiMetrics.Errors.Add(1)
-			prometheusMetrics.Responses.WithLabelValues(fmt.Sprintf("%d", code), "find").Inc()
+			app.prometheusMetrics.Responses.WithLabelValues(fmt.Sprintf("%d", code), "find").Inc()
```

Comment:
Consider using strconv.Itoa(int) instead. This module provides some conversion semantics, not string formatting.

Reply:
+1 will do in one of the next refactorings